cluster/manifests/eks-fis/01-rbac.yaml

```diff
@@ -1,12 +1,4 @@
 {{- if eq .Cluster.ConfigItems.eks_fis_support_enabled "true" }}
-# {{ range $namespace := split .Cluster.ConfigItems.eks_fis_namespaces "," }}
```

Comment:
Is the config-item `eks_fis_namespaces` used anywhere else? I see it's being removed from 2 places now.

 If not, we should also clean it up, if not in this PR then a follow-up.